packages/presets/scripts/src/declare_presets/declare_presets.cairo

```diff
@@ -0,0 +1,50 @@
+use sncast_std::ErrorData;
+use sncast_std::ProviderError::{UnknownError};
+use sncast_std::StarknetError::{ClassAlreadyDeclared};
+use sncast_std::{ScriptCommandError, ProviderError, StarknetError};
+use sncast_std::{declare, DeclareResult, get_nonce, DisplayClassHash, FeeSettings, EthFeeSettings};
+
+fn main() {
+    let max_fee = 99999999999999999;
+    let nonce = get_nonce('latest');
+
+    let account_res = declare(
+        "AccountUpgradeable",
+        FeeSettings::Eth(EthFeeSettings { max_fee: Option::Some(max_fee) }),
+        Option::Some(nonce)
+    )
+        .unwrap_err();
+    println!("{:?}", account_res);
+
+    let erc20_res = declare(
+        "ERC20Upgradeable",
+        FeeSettings::Eth(EthFeeSettings { max_fee: Option::Some(max_fee) }),
+        Option::Some(nonce)
+    )
+        .unwrap_err();
+    println!("{:?}", erc20_res);
+
+    let erc721_res = declare(
+        "ERC721Upgradeable",
+        FeeSettings::Eth(EthFeeSettings { max_fee: Option::Some(max_fee) }),
+        Option::Some(nonce)
+    )
+        .unwrap_err();
+    println!("{:?}", erc721_res);
+
+    let erc1155_res = declare(
+        "ERC1155Upgradeable",
+        FeeSettings::Eth(EthFeeSettings { max_fee: Option::Some(max_fee) }),
+        Option::Some(nonce)
+    )
+        .unwrap_err();
+    println!("{:?}", erc1155_res);
+
+    let eth_account_res = declare(
+        "EthAccountUpgradeable",
+        FeeSettings::Eth(EthFeeSettings { max_fee: Option::Some(max_fee) }),
+        Option::Some(nonce)
+    )
+        .unwrap_err();
+    println!("{:?}", eth_account_res);
+}
```

Comment:
1. Should we move max_fee to `MAX_FEE` constant?
2. It may be helpful to implement a helper function `fn declare_class(contract_name: ByteArray)`
3. I think we this script is going to fail, since the same nonce is used several times. We should fetch a new nonce before each call OR just increment it (incrementing may actually lead help to avoid "InvalidNonce" errors)

Reply:
For sure, I'm going to refactor the script and will likely do something along the lines of the provided suggestions once we can declare

Reply:
Done!